### ISSUE TITLE
[REST] Event source connection broken

### ISSUE THREAD

**Issue by egoist6** [openhab/openhab-webui#1326]
## The problem
There is a problem in the connection between MainUI and openHAB server which leads to MainUI not responding anymore to user interaction.
As per request of `J-N-K` I re-open an already existing ticket in this repo.
See history [here](https://github.com/openhab/openhab-core/issues/2686)

## Steps to reproduce
Prerequisite:
- iOS device (don't know if that problem also appears on android devices)
- unsecure connection to openhab server (http)

Connect your iOS device to openhab, start MainUI in fullscreen mode and keep the connection open for a few hours.

**Note:**
- this problem does not appear on secure connections (https)
- this problem is independent from any `oh-page` content. This was confirmed by another user who tested this with a clean `oh-page` and a single standard control.

This problem has been confirmed by approx. 5 users already (see all links in the ticket mentioned above)

## Your environment
Pi4, openhabian standard installation
This problem has been existing since OH3.1 (I think)

## Browser console
```txt
ERROR: The connection to http://<MY_OH3_SERVER>/rest/events/states was interrupted while the page was loading.
WARNING: SSE error
INFO: =!= Event source connection broken…
```
I also noticed the disconnect appears when the browser calls Google‘s safabrowsing API to this URL:
`safebrowsing.googleapis.com`
Chrome, Safari, Firefox and potentially more browsers are using this Google API

**Comment by mpmario** [comment]
I have the exact same behavior. Browser log on a mac (firefox):

> Initializing state tracking store proxy
app.3456231afb83cd720478.js:35 sseConnected false
app.3456231afb83cd720478.js:7 Start tracking states
app.3456231afb83cd720478.js:35 new SSE connection: http://10.240.0.201:8080/rest/events/states, 1 open
app.3456231afb83cd720478.js:35 [EventSource]
app.3456231afb83cd720478.js:7 Loading semantic tags ...
app.3456231afb83cd720478.js:23 Loading semantic model and building semantic homepages ...
app.3456231afb83cd720478.js:7 Successfully loaded semantic tags.
app.3456231afb83cd720478.js:7 Proxy: need HomeofficeNextDay
app.3456231afb83cd720478.js:7 updateTrackingList: No connection id, not calling the API
app.3456231afb83cd720478.js:7 Proxy: need AlarmDeactivate
app.3456231afb83cd720478.js:7 updateTrackingList: No connection id, not calling the API

**Comment by florian-h05** [comment]
I think this should have been fixed by #1807, if the issue persists, I am out of ideas.

As it only affects insecure contexts (HTTP) and there are more and more browser limitations if not using HTTPS, I would suspect the browser is not innocent here. FYI, openHAB also provides HTTPS on port 8443, just tell your browser to trust the self signed certificate.

**Comment by florian-h05** [comment]
In case the issue persists with openHAB 4.2, one may open a new issue in this repo with the sse logs from the browser console.
